spec/features/user_visits_homepage_spec.rb

```diff
@@ -5,4 +5,10 @@
     visit root_path
     expect(page).to have_css 'h1', text: 'best chess game ever'
   end
+
+  scenario 'all games listed on homepage' do
+    FactoryGirl.create(:game)
+    visit root_path
+    expect(page).to have_css('.game')
```

Comment:
I would also check that the game name appears in the text.
`expect(page).to have_css('.game', text: game.name)`